CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 Breaking changes:
 
+- [#1645](https://github.com/rails-api/active_model_serializers/pull/1645) Expanded KeyTransform into Transform. Changed :dashed transform to :dash. (@remear)
```

Comment:
Only in that it's a bit more descriptive than the current PR name of `Transforms`, but not overly verbose. It was intended as a courtesy to give brief information about the breaking changes without requiring reading the PR description and/or changes to gain any context at all. I'd be happy to change it back to match the PR name if that's what's preferred.